math/src/distribution.rs

```diff
@@ -122,6 +122,17 @@ pub trait DistributionVariance {
     }
 }
 
+impl DistributionVariance for Range<u64> {
+    fn variance(self) -> f64 {
```

Comment:
Variance of distribution is B^2 / 12 only when values are uniformly distributed in range [-B/2, B/2). Are you sure that is the case here ?

Reply:
Removed in https://github.com/gausslabs/phantom-zone/pull/37/commits/4a4850e4bac1d64a41b5635b0cf2d39c01236e69